app/src/main/java/com/android/voyageur/ui/search/PlaceDetailsScreen.kt

```diff
@@ -0,0 +1,332 @@
+              style = MaterialTheme.typography.bodySmall.copy(fontSize = 16.sp),
+              modifier = Modifier.padding(bottom = 8.dp).testTag("NoRatingsText"))
+        }
+
+        Text(
+            text =
+                if (place.priceLevel != null) "Price Level: ${"$".repeat(place.priceLevel)}"
+                else "Price Level: N/A",
+            style = MaterialTheme.typography.bodySmall.copy(fontSize = 16.sp),
+            modifier = Modifier.padding(bottom = 8.dp).testTag("PriceLevelText"))
+        Text(
+            text = "Phone: ${place.internationalPhoneNumber ?: "N/A"}",
+            style = MaterialTheme.typography.bodySmall.copy(fontSize = 16.sp),
+            modifier = Modifier.padding(bottom = 8.dp).testTag("PhoneNumberText"))
+
+        place.openingHours?.let { OpeningHoursWidget(openingHours = it.weekdayText) }
+            ?: run {
+              Text(
+                  text = "Opening hours not available",
+                  style = MaterialTheme.typography.bodySmall.copy(fontSize = 16.sp),
+                  modifier = Modifier.padding(bottom = 8.dp).testTag("NoOpeningHoursText"))
+            }
+
+        Row(
+            modifier = Modifier.fillMaxWidth().padding(top = 16.dp),
+            horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+              Button(
+                  onClick = {
+                    if (place.websiteUri != null) {
+                      val intent = Intent(Intent.ACTION_VIEW, place.websiteUri)
+                      context.startActivity(intent)
+                    } else {
+                      Toast.makeText(context, "Website not available", Toast.LENGTH_SHORT).show()
+                    }
+                  },
+                  modifier = Modifier.weight(1f).testTag("LearnMoreButton")) {
+                    Text(text = "Learn More", style = MaterialTheme.typography.labelLarge)
+                  }
+
+              Button(
+                  onClick = {
+                    if (place.latLng == null) {
+                      Toast.makeText(context, "Location not available", Toast.LENGTH_SHORT).show()
+                    } else {
+                      val gmmIntentUri =
+                          Uri.parse(
+                              "geo:0,0?q=${place.latLng.latitude},${place.latLng.longitude}(${place.name})")
+                      val mapIntent = Intent(Intent.ACTION_VIEW, gmmIntentUri)
+                      mapIntent.setPackage("com.google.android.apps.maps")
+                      context.startActivity(mapIntent)
+                    }
+                  },
+                  modifier = Modifier.weight(1f).testTag("GetDirectionsButton")) {
+                    Text(text = "Get Directions")
+                  }
+            }
+      }
+}
+
+/**
+ * Composable function that displays the opening hours of a place.
+ *
+ * @param openingHours List of opening hours for each day of the week.
+ */
+@Composable
+fun OpeningHoursWidget(openingHours: List<String>) {
```

Comment:
Here, for the opening hours I suggest you just have it one after another in separate rows, just like on google maps to have a better overview. For me I find it counterintuitive to have in a slider and have to click on another day.

Reply:
On some devices it won't fit into a single screen, that is why I have chosen this. However, this is a subjective design preference, I will leave it as is and we can discuss it later we the team